bin/schemas/story.json

```diff
@@ -0,0 +1,846 @@
+              "maximum": 255
+            },
+            "g": {
+              "type": "integer",
+              "title": "Green",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "b": {
+              "type": "integer",
+              "title": "Blue",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "a": {
+              "type": "number",
+              "title": "Alpha",
+              "minimum": 0,
+              "maximum": 1
+            }
+          }
+        }
+      }
+    },
+    "linearPattern": {
+      "type": "object",
+      "title": "Linear color pattern",
+      "required": ["type", "stops"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["none", "solid", "linear", "radial"]
+        },
+        "stops": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "#/$defs/solidPattern"
+              }
+            ],
+            "properties": {
+              "position": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+              }
+            }
+          }
+        },
+        "alpha": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "rotation": {
+          "type": "number",
+          "minimum": -360,
+          "maximum": 360
+        }
+      }
+    },
+    "pageTemplateType": {
+      "type": "string",
+      "title": "The pageTemplateType Schema",
+      "enum": [
+        "",
+        "cover",
+        "editorial",
+        "list",
+        "quote",
+        "section",
+        "table",
+        "steps"
+      ]
+    },
+    "page": {
+      "type": "object",
+      "title": "Individual Page",
+      "required": ["backgroundColor", "elements", "id", "type"],
+      "properties": {
+        "animations": {
+          "type": "array",
+          "title": "Animations on this page",
+          "items": {
+            "$ref": "#/$defs/animation"
+          }
+        },
+        "elements": {
+          "type": "array",
+          "minItems": 1,
+          "title": "The page's elements",
+          "items": {
+            "anyOf": [
+              { "$ref": "#/$defs/element" },
+              { "$ref": "#/$defs/mediaElement" },
+              { "$ref": "#/$defs/shapeElement" },
+              { "$ref": "#/$defs/stickerElement" },
+              { "$ref": "#/$defs/textElement" }
+            ]
+          }
+        },
+        "backgroundColor": {
+          "type": "object",
+          "title": "Background color",
+          "$ref": "#/$defs/pattern"
+        },
+        "defaultBackgroundElement": {
+          "type": "object",
+          "title": "The default background element",
+          "$ref": "#/$defs/shapeElement",
+          "$comment": "TODO: Move to story element or remove?"
```

Comment:
Why remove? :)

Reply:
I was wondering whether it makes sense to store the default background element in the story data JSON in the first place, since this could in theory just be a const somewhere in the editor since it's static and never changes.

From what I can see we mainly need this information about the default background element when removing the background element (e.g. an image or video) and need to restore the default.

We can still have the `isDefaultBackground` property to identify the default in the story JSON, should we ever need a data migration for it.

Does that make sense?